internal_workflow_testsuite.go

```diff
@@ -756,12 +756,15 @@ func (m *mockWrapper) getMockReturn(ctx interface{}, input []byte) mock.Argument
 	for _, arg := range reflectArgs {
 		realArgs = append(realArgs, arg.Interface())
 	}
-	found, _ := m.mock.FindExpectedCall(m.name, realArgs...)
 
-	if found < 0 {
-		// mock call not found
-		return nil
-	}
+	// There is no way to check if a mock call is expected or not. A pull request to add it was rejected. See PR:
+	// https://github.com/stretchr/testify/pull/453. We could try to call the mock method, and if the call is not
+	// expected, the mock.MethodCalled() will panic, which we would catch and just return nil from this method.
+	defer func() {
```

Comment:
Are any other situations that cause panic? My concern that some other panic is swallowed and troubleshooting is practically impossible.